content/SCALE/SCALETutorials/Apps/Customizing Advanced (Kubernetes) Settings/Configuring Host Path Safety Checks.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Configuring Host Path Validation"
+description: "This article provide information on host path validation in SCALE."
+weight: 10
+aliases:
+tags:
+- scaleapps
+---
+
+{{< toc >}}
+
+TrueNAS SCALE uses host path safety checks to ensure that host path volumes are secure when creating apps. We recommend creating datasets for applications that do not share the same host path as an SMB or NFS share.
+
+Since TrueNAS considers shared host paths non-secure, apps that use shared host paths (such as an SMB shared dataset) fail to deploy. 
+
+## Using Shared Host Paths with Safety Checks Enabled
+
+If you group share and application data under a common dataset (such as *media*) where both use a path such as */tank/media/*, the application will fail to deploy. 
```

Comment:
present tense active voice change suggestion:
```suggestion
If you group share and application data under a common dataset (such as *media*) where both use a path such as */tank/media/*, the application fails to deploy. 
```

Reply:
Forgot to `ctrl + f` for "will"!